packages/actions/src/Exports/Jobs/PrepareCsvExport.php

```diff
@@ -63,6 +63,19 @@ public function handle(): void
         $keyName = $query->getModel()->getKeyName();
         $qualifiedKeyName = $query->getModel()->getQualifiedKeyName();
 
+        if (config('database.default') === 'pgsql') {
```

Comment:
```suggestion
        /** @var Connection $databaseConnection */
        $databaseConnection = $query->getConnection();

        if ($databaseConnection->getDriverName() === 'pgsql') {
```

current logic is flawed if the model uses a non-default db connection